### PR TITLE
[VEN-2433] : VTreasuryV8 deployment on Arbitrum Sepolia

### PR DESCRIPTION
**Pull request by Debugger022**
## Description

Resolves VEN-2433

## Checklist

- [ ] I have updated the documentation to account for the changes in the code.
- [ ] If I added new functionality, I added tests covering it.
- [ ] If I fixed a bug, I added a test preventing this bug from silently reappearing again.
- [ ] My contribution follows [Venus contribution guidelines](docs/CONTRIBUTING.md).

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -119,7 +119,7 @@ jobs:
 
       - name: Export deployments
         run: |
-          for NETWORK in bsctestnet bscmainnet ethereum sepolia opbnbtestnet opbnbmainnet; do
+          for NETWORK in bsctestnet bscmainnet ethereum sepolia opbnbtestnet opbnbmainnet arbitrumSepolia arbitrumOne; do
```
Comment (chechu) [inline .github/workflows/ci.yml:122]: I would use lower case, as we are doing for the rest of the networks

File: .env.example [hunk 2]
```diff
@@ -14,6 +14,8 @@ DEPLOYER_PRIVATE_KEY=
 #ARCHIVE_NODE_ethereum=http://127.0.0.1:1248
 #ARCHIVE_NODE_opbnbtestnet=https://opbnb-testnet.nodereal.io/v1/<YOUR_KEY_HERE>
 #ARCHIVE_NODE_opbnbmainnet=https://opbnb-mainnet.nodereal.io/v1/<YOUR_KEY_HERE>
+#ARCHIVE_NODE_arbitrumOne=https://arb1.arbitrum.io/rpc
```
Comment (chechu) [inline .env.example:17]:
```suggestion
#ARCHIVE_NODE_arbitrumOne=https://open-platform.nodereal.io/<YOUR_KEY_HERE>/arbitrum-nitro/
```

To be confirmed

File: deploy/005-deploy-VTreasuryV8.ts [hunk 3]
```diff
@@ -27,6 +27,7 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     ethereum: "0x285960C5B22fD66A736C7136967A3eB15e93CC67", // ETHEREUM MULTISIG
     opbnbtestnet: "0xb15f6EfEbC276A3b9805df81b5FB3D50C2A62BDf", // OPBNBTESTNET MULTISIG
     opbnbmainnet: "0xC46796a21a3A9FAB6546aF3434F2eBfFd0604207", // OPBNBMAINNET MULTISIG
+    arbitrumSepolia: "0x1426A5Ae009c4443188DA8793751024E358A61C2", // ARBITRUM_SEPOLIA MULTISIG
```
Comment (chechu) [inline deploy/005-deploy-VTreasuryV8.ts:30]: The guardian on Arbitrum one is `0x14e0E151b33f9802b3e75b621c1457afc44DcAA0`

File: .env.example (hunk 2 above)
Comment (Debugger022) [reply .env.example:17]: That's a public url, we can use any one.

File: deploy/005-deploy-VTreasuryV8.ts (hunk 3 above)
Comment (Debugger022) [reply deploy/005-deploy-VTreasuryV8.ts]: Done https://github.com/VenusProtocol/venus-protocol/pull/456/commits/070012779c9bf4a5a306e9cdc4133c986e450b95

File: .env.example (hunk 2 above)
Comment (Debugger022) [reply .env.example]: Done https://github.com/VenusProtocol/venus-protocol/pull/456/commits/070012779c9bf4a5a306e9cdc4133c986e450b95

File: .github/workflows/ci.yml (hunk 1 above)
Comment (Debugger022) [reply .github/workflows/ci.yml]: Done https://github.com/VenusProtocol/venus-protocol/pull/456/commits/070012779c9bf4a5a306e9cdc4133c986e450b95

**Review by chechu — APPROVED** [review]

**Pull request merged**